src/Microsoft.ML.Data/Evaluators/Metrics/MultiClassClassifierMetrics.cs

```diff
@@ -57,14 +36,38 @@ public sealed class MultiClassClassifierMetrics
         /// </remarks>
         public double LogLossReduction { get; private set; }
 
+        /// <summary>
+        /// Gets the macro-average accuracy of the model.
+        /// </summary>
+        /// <remarks>
+        /// The macro-average is computed by taking the average over all the classes of the fraction
+        /// of correct predictions in this class (the number of correctly predicted instances in the class,
+        /// divided by the total number of instances in the class).
+        ///
+        /// The macro-average metric gives the same weight to each class, no matter how many instances from
+        /// that class the dataset contains.
+        /// </remarks>
+        public double MacroAccuracy { get; }
+
+        /// <summary>
+        /// Gets the micro-average accuracy of the model.
+        /// </summary>
+        /// <remarks>
+        /// The micro-average is the fraction of instances predicted correctly.
```

Comment:
>/// The micro-average is the fraction of instances predicted correctly. [](start = 8, length = 71)

The micro-average is the fraction of instances predicted correctly. The micro-average does not take class membership into account.